.github/actions/deps/action.yml

```diff
@@ -9,6 +9,14 @@ inputs:
     description: 'Path of deps-bundle.sh script'
     required: true
     default: './contrib/deps-bundle.sh'
+  compiler:
+    description: 'The compiler to use'
```

Comment:
Since this can just be `gcc` or `clang`, I'd prefer if this was a choice type instead of string.

Reply:
done